cypress/objects/loginPage.js

```diff
@@ -0,0 +1,27 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+
+    getUserFieldSelectorById() {
+        return cy.get('[id="loginForm.userId"]');
+    }
+
+    getRoleFieldSelectorById(){
+        return cy.get('[id="loginForm.role"]');
+    }
+
+    getUserOptionFromDropdown(user) {
+        return cy.get(`[aria-label="${user}"]`);
+    }
+
+    getRolesOptionFromDropdown(role) {
+        return cy.get(`[aria-label="${role}"]`);
+    }
+
+    getButtonByType(type) {
```

Comment:
This method should not be in loginPage objects as this is generic method that could be used in other pages.
You should either name this `getLoginButton` and don't pass type to it
or put this method as a helper method in `helpers.js `file
I recommend 1st option

Reply:
fixed with commit 'f073c5'